collections/_pages/smi.html

```diff
@@ -76,6 +76,9 @@
                 break;
               case "Kuma":
                 img.setAttribute("src", "assets/images/kuma.svg");
+                img.setAttribute("id", "logo-dark-light");
+                img.setAttribute("data-logo-for-dark", "assets/images/kuma-icon-white.svg");
+                img.setAttribute("data-logo-for-light", "assets/images/kuma.svg");
```

Comment:
Can we think of better name of image? 
So as to maintain consistency with the image for "light theme"? 
Maybe we can changes both names as `dark-kuma` and `light-kuma`? or any other suggestions? 

Reason being if any developer comes to the codebase, how he will get to know which one is for light theme and which one is for dark? 
For line number 80, naming convention seems good but I think we should change for line number 81 🤔

Reply:
Sure! 
I will change the names as `kuma-icon-white` for dark theme and `kuma-icon-dark` for light theme. This seems more descriptive?

Reply:
Yes, looks fine now 🚀